app/controllers/movies_controller.rb

```diff
@@ -0,0 +1,31 @@
+class MoviesController < ApplicationController
+  def index
+
+    params[:sort] ? (sort = params[:sort].to_sym) : sort = :id
```

Comment:
this line is great! I'm a fan!

Ternaries also work so you can assign things like this (this does the same thing you did):
```ruby
sort = params[:sort] ? params[:sort].to_sym : :id
```